linode_api4/objects/base.py

```diff
@@ -216,6 +233,7 @@ def _populate(self, json):
 
         # hide the raw JSON away in case someone needs it
         self._set('_raw_json', json)
+        self._set('_updated', False)
```

Comment:
why do we need to set `updated` to be false here?

Reply:
Good question! When `_populate` is called all local fields are overwritten with the contents of the `json` argument, so we need to set `_updated` to false to effectively invalidate any changes made by the user.